ReleaseNotes-2.8

```diff
@@ -39,6 +39,8 @@ Features
  * /about route is now independent from request format
  * workers are now also tracked when they went away (new states "down", "away" and "dead")
  * Main projects list is now filtered based on a configurable (by the admin) regular expression
+ * Users can download the public key and SSL certificate for a project via the project home page
+ * New API route to get public key and SSL certificate: GET /source/:project_name/:_keyinfo
```

Comment:
The route is correct?, I mean the `_keyinfo` is without `:` ? GET /source/:project_name/_keyinfo